packages/react/src/file-input/docs/overview.mdx

```diff
@@ -70,3 +71,25 @@ Use the `invalid` prop to indicate if the user input is invalid.
 ```jsx live
 <FileInput label="Invalid" invalid message="This input is invalid" />
 ```
+
+## Accept
+
+Use the `accept` prop to restrict the accepted file types. This will automatically add a hint to guide users, although you can override this by providing your own `hint`.
+
+_NOTE: for security reasons you must validate the file type in your backend and not rely on frontend validation._
```

Comment:
Might be worth adding that it is only a suggestion of restriction. Users can still do whatever they want

Reply:
As in, not add the `accept` attribute?

Reply:
Oh, do you mean, that it sets a mutable default filter in your file picker window?

Reply:
Yeah. You can also drag and drop from another window and it will obviously not be restricted

Reply:
- [ ] explain guidance vs restriction